ci/before_install.sh

```diff
@@ -87,7 +87,10 @@ msys_install() {
   "
   pacman --noconfirm -S --needed ${packages}
   pacman --noconfirm -U \
-    http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-libbotan-2.13.0-1-any.pkg.tar.xz
+    http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-libbotan-2.13.0-1-any.pkg.tar.xz || \
```

Comment:
I suggest to introduce an environment variable for botan version and re-use it in `cmake` dependency too,
As we have an outstanding task #1279

Reply:
@rrrooommmaaa This PR is just to quick-fix msys2 CI, more complicated things should be done as part of that task. And we hardcode version 2.13 for botan here due to this problem: https://github.com/rnpgp/rnp/issues/1104#issuecomment-623171383

Anyway, it should be more clean to use variable for botan's package name instead of duplicating it, please see the updated PR.